data.json

```diff
@@ -593,6 +593,15 @@
     "dateAdded": "2019-10-04T16:13:00.000Z",
     "tags": ["hacktoberfest", "stickers", "accessories", "expired"]
   },
+  {
+    "name": "Vonage",
+    "difficulty": "medium",
+    "description": "Make a pull request to issues tagged with \"Hacktoberfest\" label and in one of Vonage repos. Check project repos here: https://github.com/Vonage/. After you create PR submit form and get $5 gift card or socks",
+    "reference": "https://developer.nexmo.com/hacktoberfest",
+    "image": "https://res.cloudinary.com/dsg3ucw55/image/upload/v1599671200/Logo_nstaby.svg",
+    "dateAdded": "2020-10-01T10:15:00.000Z",
+    "tags": ["hacktoberfest", "clothing", "credit/coupon/gift-card"]
```

Comment:
```
    "tags": ["hacktoberfest", "clothing", "credit/coupon/gift-card"]
```
@vikaspotluri123 Which tag should we use for gift-card from your opinion?

Reply:
Since it's for Open Collective, maybe `donation`?